numpy/core/code_generators/generate_umath.py

```diff
@@ -912,6 +912,12 @@ def english_upper(s):
           TD(notimes_or_obj),
           signature='(n?,k),(k,m?)->(n?,m?)',
           ),
+'sinc' :
+    Ufunc(1, 1, None,
+          docstrings.get('numpy.core.umath.sinc'),
+          None,
+          TD(inexact, f='sinc', astype={'e':'f'}),
```

Comment:
I think the file does this commonly, so you can keep it, but a space after the `:` would be nicer.